Dockerfile

```diff
@@ -24,5 +24,12 @@ RUN python manage.py collectstatic --noinput
 RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
 RUN apt-get install -y nodejs
 
+# Install JS dependencies
+RUN npm install
+RUN npm install babel-core@^7.0.0-bridge.0 --save-dev --dev jest --save-dev vue-jest vue-template-compiler --save-dev jest-serializer-vue -g yarn
```

Comment:
Shouldn't need this line as in the .travis.yml comment.

Reply:
deleted line

Reply:
for whatever reason deleting this line causes line 21 of .travis.yml (  - docker-compose run web jest -u ) to throw this error:

Error response from daemon: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"jest\": executable file not found in $PATH": unknown

and adding it back in makes it all good again, probably don't have to install all these packages at once? But it's getting late and I'll have to figure that out later. 

Taking the line out of .travis.yml was causing the build to fail. 

Maybe I'm not doing something right but for now these lines (or something in them) are enabling my stuff to work.